### PR TITLE
use fuzzy instead of buildjet

### PR DESCRIPTION
**Pull request by LesnyRumcajs**
**Summary of changes**
Changes introduced in this pull request:
- use `fuzzy` instead of `buildjet`. It shouldn't require much of its resources and frees us completely from `buildjet` bills.

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

**Review by lemmih — APPROVED** [review]

**Comment by lemmih** [comment]
`forest` now has two runners, and `fil-actor-states` has one.

**Review by hanabi1224 — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
   forest-sync-check:
     if: ${{ contains(github.event.pull_request.labels.*.name, 'Release') }}
     name: forest calibnet sync check
-    runs-on: buildjet-8vcpu-ubuntu-2004
+    runs-on: fuzzy
```
Comment (hanabi1224) [inline .github/workflows/tests.yml:51]: We should disable `SCCACHE_GHA_ENABLED` on `fuzzy` and use local FS as cache backend maybe

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -15,7 +15,6 @@ on:
       - main
 
 env:
-  SCCACHE_GHA_ENABLED: "true"
```
Comment (hanabi1224) [inline .github/workflows/tests.yml:18]: Hmm I think jobs that run on GH-hosted runners still need it

Comment (LesnyRumcajs) [reply .github/workflows/tests.yml:18]: Indeed. I am too trigger-happy.

Comment (LesnyRumcajs) [reply .github/workflows/tests.yml]: It should be fine now.

**Review by lemmih — APPROVED** [review]

**Pull request merged**